wandb/sdk/wandb_run.py

```diff
@@ -291,7 +291,7 @@ def __init__(self, config=None, settings=None):
 
         # Initial scope setup for sentry. This might get changed when the
         # actual run comes back.
-        sentry_set_scope("user", self._entity, self._project)
+        sentry_set_scope("user", self._entity, self._project, self._settings.email)
```

Comment:
use kwargs when there are this many args please.

Reply:
You mean like pack them into a dictionary and use the `**` operator?

Reply:
Sorry, more precisely, keyword arguments

I was thinking at least:
```
sentry_set_scope("user", self._entity, self._project, email=self._settings.email)
```
but you might want to use entity=self._entity, project=self._project also.

I think as soon as you get over 2 or 3 arguments, i think positional arguments are too error prone.   And definitely if you have more than one optional argument, positional arguments should never be used.